docker-compose/docker-compose.test.postgres.yml

```diff
@@ -0,0 +1,151 @@
+version: "3.3"
+services:
+  test-consumer-api:
+    extends: 
+      file: docker-compose.test.yml
+      service: test-consumer-api
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Local
+      - ASPNETCORE_URLS=http://0.0.0.0:80
+
+      - Modules__Files__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=files;Password=Passw0rd;Port=5432
+      - Modules__Files__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Messages__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=messages;Password=Passw0rd;Port=5432
+      - Modules__Messages__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Relationships__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=relationships;Password=Passw0rd;Port=5432
+      - Modules__Relationships__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Synchronization__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=synchronization;Password=Passw0rd;Port=5432
+      - Modules__Synchronization__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Tokens__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=tokens;Password=Passw0rd;Port=5432
+      - Modules__Tokens__Infrastructure__SqlDatabase__Provider=Postgres
+
+      - Modules__Devices__Infrastructure__SqlDatabase__Provider=Postgres
+      - Modules__Devices__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=devices;Password=Passw0rd;Port=5432
+
+      - Modules__Challenges__Infrastructure__SqlDatabase__Provider=Postgres
+      - Modules__Challenges__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=challenges;Password=Passw0rd;Port=5432
+
+      - Modules__Quotas__Infrastructure__SqlDatabase__Provider=Postgres
+      - Modules__Quotas__Infrastructure__SqlDatabase__ConnectionString=Server=postgresql;Database=enmeshed;User Id=quotas;Password=Passw0rd;Port=5432
+    depends_on:
+      test-seed-postgresql:
+        condition: service_completed_successfully
+      test-rabbitmq:
+        condition: service_started
+      test-azure-storage-emulator:
+        condition: service_started
+
+  test-admin-ui:
+    extends: 
+      file: docker-compose.test.yml
+      service: test-admin-ui
```

Comment:
Hmm... I don't know this feature. But it seems that this limits your abilities compared to what I had in mind.

So instead of this, consider the following: `docker compose -f docker-compose.test.yml -f docker-compose.test.postgres.yml`. This combines all services in both files.